services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +43,62 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
```

Comment:
check if event is NULL and that queue has been created

Reply:
Need to check that the queue isn't null